benchmark/attention_inference/run_all.sh

```
@@ -0,0 +1,32 @@
+#     config = subset of configs (default: all six)
+# e.g. run_all.sh rtx_pro_6000 GPU-41233e81-...   |   run_all.sh b300 "" llama
+set -uo pipefail
+
+ARCH=${1:?usage: run_all.sh <arch_label> [gpu] [config ...]}
+if [ -n "${2:-}" ]; then
+    export CUDA_VISIBLE_DEVICES=$2
+fi
+shift; [ $# -gt 0 ] && shift
+CONFIGS=("$@")
+[ ${#CONFIGS[@]} -eq 0 ] && CONFIGS=(llama qwen35 gpt_oss deepseek_v4 kimi_k3 auto_regressive_dit)
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)